src/CMakeLists.txt

```diff
@@ -48,13 +48,13 @@ if (TILES)
                 ${MESSAGES_CPP}
                 ${RESOURCE_RC})
         if(MSYS2)
-            # /MANIFESTINPUT not supported by GNU linker
-            # Defaul embed via `windres.exe` compiling `resource.rc` -> `resource.rc.obj`
+            target_link_options(cataclysm-tiles PRIVATE
+                "LINKER:--strip-debug"
```

Comment:
Why is this needed here? Debug symbols are already determined by the build type.